packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -279,24 +279,25 @@ function validateSheets(sheetsRegistry: typeof SheetsRegistry)
 const enableDevelopmentEmailsSetting = new DatabaseServerSetting<boolean>('enableDevelopmentEmails', false)
 async function sendEmail(renderedEmail: RenderedEmail): Promise<boolean>
 {
-  if (process.env.NODE_ENV === 'production' || enableDevelopmentEmailsSetting.get()) {
+ 
+  // if (process.env.NODE_ENV === 'production' || enableDevelopmentEmailsSetting.get()) {
```

Comment:
commenting this out is unnecessary; you can just append `--production` to `yarn start-prod-db`